packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1033,8 +1033,6 @@ oie.enroll.okta_verify.setup.email.title = Check your email
 oie.enroll.okta_verify.setup.sms.title = Check your text messages
 oie.enroll.okta_verify.setup.skipAuth.subtitle = To set up Okta Verify on additional devices, you can copy an existing Okta Verify account onto a new device.
 # {0} is a enrolled device's name
-oie.enroll.okta_verify.setup.skipAuth.openOv = Open Okta Verify on any of your other Okta Verify devices (<$1>{0}</$1>).
```

Comment:
Was there a corresponding monolith change that removed this i18n key from monolith?

Reply:
no, there is none

Reply:
Can you add this i18n key back in? This file needs to be in sync with that's in monolith. It's fine the string is no longer being used

Reply:
Makes sense. Added: https://github.com/okta/okta-signin-widget/pull/3362/commits/ae6ecf74f07619f16b35c67cd41f91143cb6472a